source/agora/node/main.d

```diff
@@ -322,3 +303,61 @@ shared static this ()
         Address("http://boa1xrval7gwhjz4k9raqukcnv2n4rl4fxt74m2y9eay6l5mqdf4gntnzhhscrh.validators.testnet.bosagora.io/"),
     ];
 }
+
+/// Helper function for TestNet configuration
+private Nullable!Config makeTestNetConfig (AgoraCLIArgs cmdln)
+{
+    import agora.api.FullNode;
+    import agora.common.DNS;
+    import vibe.web.rest;
+    static import std.file;
+    import core.time;
+
+    // If the user provided a configuration, they likely want to run
+    // a validator, or have some special config (e.g. interfaces, logging...).
+    // In this case, we just override the `consensus` part, and set `network`
```

Comment:
should we `ensure(config.consensus == ConsensusConfig.init)` before overriding or log the new consensus config?

Reply:
I decided against the former because this option is for non-technical people so it should "just work".
As for the later, we've been logging the config since forever in Faucet and it hasn't been useful a single time. I added a logging statement when I tested it to make sure things worked as expected, but I don't think a permanent one is going to be more useful than in Faucet, unless you feel strongly about it.

Reply:
Faucet's config is static tho, this one is dynamically fetched and if a user wants to look up what config they are running they have no means to do that, right?